bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/protocol/googletv/GoogleTVConnectionManager.java

```diff
@@ -465,8 +465,8 @@ private void initialize() {
         config.reconnect = (config.reconnect > 0) ? config.reconnect : DEFAULT_RECONNECT_SECONDS;
         config.heartbeat = (config.heartbeat > 0) ? config.heartbeat : DEFAULT_HEARTBEAT_SECONDS;
         config.delay = (config.delay < 0) ? 0 : config.delay;
-        config.shim = (config.shim) ? true : false;
-        config.shimNewKeys = (config.shimNewKeys) ? true : false;
+        config.shim = config.shim;
```

Comment:
What is the point of setting the var to itself? I think this can be removed completely, or was something else intended?
Same for next line.

Reply:
That is what we call lack of caffeine.  You're absolutely right.  In the other PR I was creating a new boolean that I was using, in this case we can get rid of them entirely.